ichnaea/api/submit/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+import pytest
+
+# Use pytest rich asserts in a file that doesn't match test_* pattern
+pytest.register_assert_rewrite("ichnaea.api.locate.tests.base")
```

Comment:
I think this one needs to be `ichnaea.api.submit.tests.base`.

Reply:
🤦‍♂ yep copy and paste gets me again